.github/workflows/cherryPick.yml

```diff
@@ -43,12 +43,19 @@ jobs:
         with:
           GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
 
+      - name: Get previous app version
+        id: getPreviousVersion
+        uses: Expensify/App/.github/actions/javascript/getPreviousVersion@main
+        with:
+          SEMVER_LEVEL: 'PATCH'
+
+      - name: Fetch history of relevant refs
+        run: |
+          git fetch origin main staging --no-tags --shallow-exclude ${{ steps.getPreviousVersion.outputs.PREVIOUS_VERSION }}
+
       - name: Get version bump commit
         id: getVersionBumpCommit
         run: |
-          # Find the commit for the version-bump
-          # This command fetches commits in the last hour on the main branch
-          git fetch origin main --no-tags --shallow-since="$(( $(date +%s) - 3600 ))"
```

Comment:
Do we want to also update https://github.com/Expensify/App/blob/e812eebd0f43/.github/workflows/README.md#L63 because it's no longer really relevant?

Reply:
Thanks! And yes I meant specifically that line :)